baybe/searchspace/discrete.py

```diff
@@ -646,40 +651,67 @@ def get_candidates(
 
     def transform(
         self,
-        data: pd.DataFrame,
+        df: pd.DataFrame | None = None,
+        /,
+        *,
+        allow_missing: bool = False,
+        allow_extra: bool | None = None,
+        data: pd.DataFrame | None = None,
     ) -> pd.DataFrame:
-        """Transform parameters from experimental to computational representation.
+        """See :func:`baybe.searchspace.core.SearchSpace.transform`."""
+        # >>>>>>>>>> Deprecation
+        if not ((df is None) ^ (data is None)):
```

Comment:
All of this deprecation stuff is identical code at several places, is there no easy/easier way to avoid all of this boilerplate?

Reply:
I know its duplicated but at this stage I honestly didn't care since it's already clear from the get go that it won't remain in the code... 😬 So I suggest we just keep as is, unless you have strong opinion against (but then please suggest an alternative 🙃 )

Reply:
No, I will just use the argument "boilerplate is fine if we know stuff is only temporary" in the future and then refer to this discussion if there are complaints ;)